lib/katello/repository_types/README.md

```diff
@@ -1,64 +0,0 @@
-# Registering a new repository type
```

Comment:
Should we update this file for pulp3 content type registration?

Reply:
yeah not a bad idea. I didn't realize it got deleted.